test/jdk/java/awt/ColorClass/AlphaColorTest.java

```diff
@@ -84,8 +84,8 @@ public void paint(Graphics g) {
         Color color = new Color(255, 255, 255, 127);
         frame.add("Center", new AlphaColorTest(color));
         frame.setUndecorated(true);
-        frame.setLocationRelativeTo(null);
         frame.pack();
+        frame.setLocationRelativeTo(null);
         frame.setVisible(true);
```

Comment:
That was a "typo" to place the window to the center of the screen we need to pack/setSize/setBounds and then setLocationRelativeTo